BREAKING.md

```diff
@@ -108,32 +91,32 @@ More details can be found on `IIntervalCollectionEvent`'s doc comment.
 See previous ["Upcoming" change notice](#bindToContext-to-be-removed-from-IFluidDataStoreChannel) for info on how this removal was staged.
 
 ### MergeTree class no longer exported
-    The MergeTree class was deprecated and is no longer be exported. This should not affect usage as MergeTree is an internal class, and the public API exists on the Client class, which will continue to be exported and supported.
+The MergeTree class was deprecated and is no longer be exported. This should not affect usage as MergeTree is an internal class, and the public API exists on the Client class, which will continue to be exported and supported.
 
 ### Marker.toString simplified
 
 In merge-tree, Marker's string representation returned by `toString` was simplified.
```

Comment:
Is this breaking change? The statement here should be around "breaking" element of the change and what user needs to do to accommodate for the change.  We should do one more brownbag on this topic.

Reply:
Definitely agree we should do another brownbag. Although this might technically be a breaking change, we should only add items to BREAKING.md if there is some action our users can take.